src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let man = people.filter(p => p.sex === 'm');
+
+  if (century) {
+    man = people.filter(p => p.sex === 'm'
+    && century === Math.ceil(p.died / 100));
```

Comment:
but you already have all `men` filtered out
```suggestion
    men = men.filter(man => century === Math.ceil(man.died / 100));
```